File: pios.projekt.API/pios.projekt.models/Models/TimetableRow.cs
namespace pios.projekt.models.Models
{
    public class TimetableRow
    {
        public int Id { get; set; }
		public int FromHour { get; set; }
        public int ToHour { get; set; }
        public TimetableItem Monday { get; set; }
        public TimetableItem Tuesday { get; set; }
        public TimetableItem Wednesday { get; set; }
        public TimetableItem Thursday { get; set; }
        public TimetableItem Friday { get; set; }
    }
}

File: pios.projekt.front/src/Models/TimetableRow.ts
import { TimetableItem } from './TimetableItem';

export interface TimetableRow {
    id: number;
    classroomId: number;
    fromHour: number;
    toHour: number;
    monday: TimetableItem | null;
    tuesday: TimetableItem | null;
    wednesday: TimetableItem | null;
    thursday: TimetableItem | null;
    friday: TimetableItem | null;
}

export class TimetableRowClass implements TimetableRow {
    id = 0;
    classroomId = 0;
    fromHour = 0;
    toHour = 0;

    monday = null;
    tuesday = null;
    wednesday = null;
    thursday = null;
    friday = null;
}
